hooks/conan-center.py

```diff
@@ -558,6 +558,10 @@ def test(out):
     @run_test("KB-H053", output)
     def test(out):
         def _is_private_import(line):
+            full_line_allowed_list = ["from conans.model import Generator"]
+            if line in full_line_allowed_list:
+              return true
+            
             allowed_list = ["tools", "errors"]
```

Comment:
```suggestion            
            allowed_list = ["tools", "errors", "model.Generator"]
```
@uilianries what do you think of this one ?

Reply:
sounds better!